package.json

```diff
@@ -40,7 +40,7 @@
     "lint:interactive": "eslint-interactive './packages/**/*.{js,ts,jsx,tsx}'",
     "format": "prettier --check './packages/**/*.{js,ts,jsx,tsx}'",
     "format:fix": "prettier --write './packages/**/*.{js,ts,jsx,tsx}'",
-    "codesandbox:insall": "npm i -g pnpm@8 & pnpm i --no-frozen-lockfile",
+    "codesandbox:insall": "npm i -g pnpm@8 && pnpm i --no-frozen-lockfile",
```

Comment:
`&` runs both sides concurrently. `&&` runs right-hand after left-hand succeed.

Reply:
Oops 😇
Thank you!